app/controllers/provider_interface/provider_users_invitations_controller.rb

```diff
@@ -39,10 +39,22 @@ def update_providers
     end
 
     def edit_permissions
-      @wizard = ProviderUserInvitationWizard.new(session, current_step: 'permissions', current_provider_id: params[:provider_id])
+      @provider = Provider.find(params[:provider_id])
+      @wizard = ProviderUserInvitationWizard.new(session, current_step: 'permissions', current_provider_id: @provider.id)
       @wizard.save_state!
 
-      @permissions_form = ProviderUserPermissionsForm.new(@wizard.permissions_for(params[:provider_id]))
+      permission_struct = Struct.new(:slug, :name)
+      @permissions = [
+        permission_struct.new('manage_users', 'Manage users'),
+        permission_struct.new('make_decisions', 'Make decisions'),
+      ]
+
+      permissions_form_struct = Struct.new(:id, :provider_id, :permissions)
+      @permissions_form = permissions_form_struct.new(
+        @provider.id,
+        @provider.id,
```

Comment:
Is it correct that we have `@provider.id` twice here? Perhaps this change needs another look, the creation of `Structs` inside the method seems like something that could be cleaned up.

Reply:
Talked about this on slack, this is a bit gnarly but generally harmless. Refactor later!

Reply:
Yes, this is best handled as a wizard refactor.